src/fake_bpy_module/generator/writers.py

```diff
@@ -137,14 +137,14 @@ def _write_function_code(self, func_node: FunctionNode):
             if not dtype_list_node.empty():
                 dtype_nodes = find_children(dtype_list_node, DataTypeNode)
                 if len(dtype_nodes) >= 2:
-                    dtype_str = f"typing.Union[{', '.join([n.to_string() for n in dtype_nodes])}]"
+                    dtype_str = ' | '.join(n.to_string() for n in dtype_nodes)
                 else:
                     dtype_str = dtype_nodes[0].to_string()
```

Comment:
```
                if len(dtype_nodes) >= 2:
                    dtype_str = ' | '.join(n.to_string() for n in dtype_nodes)
                else:
                    dtype_str = dtype_nodes[0].to_string()
```

I think above code can be changed to a simpler code.
Same for others.

```
                dtype_str = ' | '.join(n.to_string() for n in dtype_nodes)
```

Reply:
Indeed